other/check-smf-languages.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * Simple Machines Forum (SMF)
+ *
+ * @package SMF
+ * @author Simple Machines https://www.simplemachines.org
+ * @copyright 2020 Simple Machines and individual contributors
+ * @license https://www.simplemachines.org/about/smf/license.php BSD
+ *
+ * @version 2.1 RC3
+ */
```

Comment:
I don't understand full was the intention of this test is,
maybe a description would help?